ACKategoriesCore/PublisherExtensions.swift

```diff
@@ -0,0 +1,11 @@
+import Combine
+
+@available(macOS 10.15, *)
```

Comment:
We should take care about iOS as well, actually pipeline failed on that and also I am pretty sure, that this won't be available through Carthage as I don't see any change to its project file 🤔